src/huggingface/fortex/huggingface/bio_ner_predictor.py

```diff
@@ -191,15 +197,15 @@ def pack(
         self,
         data_pack: DataPack,
         output_dict: Optional[Dict[str, Dict[str, List[Any]]]] = None,
+        context: Optional[Annotation] = None,
     ):
         """
         Write the prediction results back to datapack. by writing the predicted
-        ner to the original subwords and convert predictions to something that
+        ner to the context and convert predictions to something that
```

Comment:
We simply say something like write predicted results by aggregating subwords into named entity mentions

Reply:
fixed it.